optaplanner-wb-screens/optaplanner-wb-solver-editor/optaplanner-wb-solver-editor-backend/src/test/resources/ProjectA/src/main/java/org/projectA/MySolution.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2016 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.projectA;
+
+
+import java.util.Collection;
+
+import org.optaplanner.core.api.domain.solution.PlanningSolution;
+import org.optaplanner.core.api.domain.solution.Solution;
+import org.optaplanner.core.api.score.buildin.hardsoft.HardSoftScore;
+
+@PlanningSolution
+public class MySolution
```

Comment:
To avoid any trouble later on (NPE's on get PRoblemFacts etc) it's probably best to copy-paste

 https://github.com/droolsjbpm/optaplanner/blob/master/optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/TestdataSolution.java
https://github.com/droolsjbpm/optaplanner/blob/master/optaplanner-core/src/test/java/org/optaplanner/core/impl/testdata/domain/TestdataEntity.java
etc